staff/maryam-malek/app-template/contact.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Contact</title>
+</head>
+<body>
+    <header>
+        <h2>Contact Us</h2>
+    </header>
+    <section>
+        <h3>Send us a message!</h3>
+        <form action="" method="get">
```

Comment:
Switch to the method POST, because it is a form to send data... POST = Send data to server & GET = receive data from the server

Reply:
POST sends the data inside the body of the HTML whereas GET sends the data via URL. Both ways are used to communicate with the server, and since we are not using server-side content, there's no need to care for the method attribute (just wanted to clarify)